src/filesys.cpp

```diff
@@ -359,8 +356,9 @@ std::string TempPath()
 		compatible with lua's os.tmpname which under the default
 		configuration hardcodes mkstemp("/tmp/lua_XXXXXX").
 	*/
+
 #ifdef __ANDROID__
-	return g_settings->get("TMPFolder");
+	return porting::path_cache;
 #else
 	return DIR_DELIM "tmp";
```

Comment:
this will 100% not work on Windows

Reply:
This is in the Linux section. Windows is elsewhere